src/NServiceBus.AcceptanceTesting/NServiceBus.AcceptanceTesting.csproj

```diff
@@ -13,7 +13,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="NUnit" Version="3.7.1" />
```

Comment:
Wasn't entirely sure about this one. But since it is also used as a nuget package I figured it makes sense to align with other generated packages that expose NUnit as a reference

Reply:
Yeah, this should have been like the other non-private references since we do build a package for this.